docs/source/wakepy-mode-lifecycle.md

```
@@ -141,13 +166,14 @@ long_running_task() # activates and deactivates
 ```
 
-The python process creates a new Mode context manager instance and uses it every time a decorated function is called. If the process enters a *Activation Failed* state, the *Action on Fail* occurs (See: {numref}`fig-mode-activity-diagram`). This action may be an exception, a warning, or a custom action as determined by the [`on_fail`](#wakepy.keep.running) input parameter.
+The python process creates a new Mode context manager instance and uses it every time a decorated function is called. If the process enters a *Activation Failed* state, the *Action on Fail* occurs (See: {numref}`fig-mode-activity-diagram`). This action may be an exception, a warning, or a custom action as determined by the {func}`on_fail <wakepy.keep.running>` input parameter.
 
 
 The {numref}`fig-activate-mode-activity-diagram` presents an activity diagram from the "Activate Mode" step of {numref}`fig-mode-activity-diagram`. The steps are:
 - ***Check WAKEPY_FAKE_SUCCESS***: If the [`WAKEPY_FAKE_SUCCESS`](#WAKEPY_FAKE_SUCCESS) environment variable is set to a truthy value, the `WakepyFakeSuccess` method is inserted at the beginning of the methods list. This special method has `.caniuse()`, `.enter_mode()`, `.heartbeat()` and `.exit_mode()` which always succeed without making any real system calls, which is useful for testing.
-- ***Prioritize Methods***: Methods are prioritized first with `methods_priority` from the user, if given. Then `Method.supported_platform` is compared to the current platform, and unsupported Methods are placed last in the prioritized list.
-- ***Activate with a Method***: Try to activate the Mode using the Method with highest priority. This is explained in more detail in the [next section](#section-activating-with-a-method). Note that only *one* Method is ever used to activate a Mode; the first one which does not fail, in priority order. Unsupported methods (at the end of the list) are not tried at all.
+- ***Prioritize Methods***: Methods are prioritized with `methods_priority` from the user, if given.
+- ***Platform Support***: Methods not supported by the current platform are removed from the list of methods to be tried. (Check current platform against `Method.supported_platform`) 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)